pkgs/applications/editors/jetbrains/readme.md

```diff
@@ -35,31 +35,28 @@ fetchurl {
 }
 ```
 
-## How to update stuff:
- - Run ./bin/update_bin.py, this will update binary IDEs, and automatically commit them
- - Source builds need a bit more effort, as they **aren't automated at the moment**:
-   - Run ./source/update.py ./source/sources.json. This will update the source version to the latest available version.
-   - Run these commands respectively:
-     - `nix build .#jetbrains.idea-oss.src.src && ./source/build_maven.py source/idea_maven_artefacts.json result/` for IDEA
-     - `nix build .#jetbrains.pycharm-oss.src.src && ./source/build_maven.py source/pycharm_maven_artefacts.json result/` for PyCharm
-   - Make sure the Kotlin version used is correct.
-     - Check the recommended Kotlin version in `.idea/kotlinc.xml` in the IDEA source root
-   - Do a test build
-   - If it succeeds, make a commit
-   - make a PR/merge
-   - If it fails, ping/message GenericNerdyUsername or the nixpkgs Jetbrains maintainer team
+## How to update IDEs:
+ - Run `./updater/main.py`, this will update binary and source IDEs. After this you can commit them
+   - See `./updater/main.py --help` for additional flags.
+   - The IDEs have `passthru.updateScript` set up to run `./updater/main.py`. The script then uses the `UPDATE_NIX_*`
+     environment variables as documented.
```

Comment:
Nit: one sentence per line:

```suggestion
 - Run `./updater/main.py`, this will update binary and source IDEs.
   After this you can commit them.
   - See `./updater/main.py --help` for additional flags.
   - The IDEs have `passthru.updateScript` set up to run `./updater/main.py`.
     The script then uses the `UPDATE_NIX_*` environment variables as documented.
```